scripts/main.js

```diff
@@ -0,0 +1,35 @@
+const hamburger = document.querySelector('.hamburger');
+ const menu = document.querySelector('.menu_navigation');
+ const menuLinks = document.querySelectorAll('.bar_navigation');
+ const btnOpenMenu = document.querySelector('.iconemenu');
+const btnCloseMenu = document.querySelector('.btn-close');
+
+const isActiveClassExisted = (element) => element.classList.contains('active');
+
+const refreshIcon = () => {
+  if (isActiveClassExisted(menu)) {
+    hamburger.setAttribute('src','images/hamb-close.png');
+  } else {
+    hamburger.setAttribute('src','images/iconemenu.png');
+  }
+};
+const openMenu = () => {
+  menu.classList.toggle('active');
+  refreshIcon();
+};
+
+ const closeMenu = () => {
+  menu.classList.remove('active');
+  refreshIcon();
+  
+};
+
+hamburger.addEventListener('click', openMenu);
+menuLinks.forEach((menuLink) => {
+  menuLink.addEventListener('click', closeMenu);
+});
```

Comment:
- [ ] While trying to implement the required changes from the previous reviewer, some parts of the code have been broken.
1. The faded blue background is no more displayed.
2. The (X) icon should not be added as a separate element, it should be toggle with the hamburger icon when it is clicked. When you clicked the hamburger icon, it should change to (X) symbol, and when that is clicked, the mobile menu options should disappeared. 
3. When a mobile menu option is clicked, an appropriate part of the page should displayed and the mobile menu options disappear. Currently, your app is not behaving that way. When a mobile menu option is clicked, the remaining options still remain on the page which is contrary to the requirement for that part in the project. Kindly look into it and fix the issue. Please take a look :point_down: 

[your-page.webm](https://user-images.githubusercontent.com/26597432/204005795-2f14caf2-b593-4913-9ed4-10762dcba379.webm)

- [ ] Below is the guiding template that shows how that particular section of your project should look like. :point_down: 

![temp](https://user-images.githubusercontent.com/26597432/204006014-28541d21-69cd-4328-9bb2-ce4a38a79db0.png)